examples/terraform/openstack/control_plane.tf

```diff
@@ -14,6 +14,11 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+resource "openstack_compute_servergroup_v2" "control_plane" {
+  name     = "${var.cluster_name}-control_plane"
+  policies = [var.control_plane_group_policy]
+}
```

Comment:
Are policies supported by default on all OpenStack installations or does it require policies to be explicitly enabled and created?